packages/playwright-test/src/matchers/toMatchSnapshot.ts

```diff
@@ -178,7 +178,7 @@ class SnapshotHelper<T extends ImageComparatorOptions> {
     if (isWriteMissingMode) {
       writeFileSync(this.snapshotPath, actual);
       writeFileSync(this.actualPath, actual);
-      this.testInfo.attachments.push({ name: addSuffixToFilePath(this.snapshotName, '-actual'), contentType: this.mimeType, path: this.actualPath });
+      this.testInfo._attachmentsPush({ name: addSuffixToFilePath(this.snapshotName, '-actual'), contentType: this.mimeType, path: this.actualPath });
```

Comment:
Is this to avoid an "attach screenshot-actual.png" step? If so, why are there no test changes?

Reply:
This is to remove the dupes